### ISSUE TITLE
Training log

### ISSUE THREAD

**Issue by DeepRunner** [zyang-ur/onestage_grounding#2]
Hi,
Could you please share the training log?
It seems my training  with a small drop of accuracy on Flickr, compared with official result(67% -> 63%)?
I use 4  GPUs(1080), with learning rate of 4*10e-4.
Thanks

**Comment by zyang-ur** [comment]
One possibility could be the batch size on multiple GPUs, as my implementation didn't include sync BN. Maybe try single CPU with batch 32? I'll try to locate the logs :)
